src/mtcp/mtcp_util.c

```diff
@@ -42,10 +42,10 @@
 #include "mtcp_util.h"
 #include "../membarrier.h"
 
-unsigned long mtcp_strtol (char *str)
+unsigned long long mtcp_strtoll (const char *str)
```

Comment:
We seem to have a mixed formatting convention in this file.  Sometimes it's `FUNCTION (arg)` and dometimes it's `FUNCTION(arg)`.  It's optional, but you might want to consider an extra commit that cleans this up everywhere:
   `FUNCTION (arg)` ->  `FUNCTION(arg)`